src/CMakeLists.txt

```diff
@@ -39,6 +39,15 @@ find_package(OpenSSL REQUIRED)
 
 include(GenerateExportHeader)
 
+if(MGCLIENT_ON_WINDOWS)
+    set(LIBS_ON_WINDOWS ws2_32)
```

Comment:
Did you consider adding `MGCLIENT_` prefix to both `LIBS_ON_WINDOWS` and `LINK_OPTIONS_ON_WINDOWS` since these are the variables related to `mgclient`?

Reply:
I haven't considered it, but as you mentioned it is an obvious thing to do.